src/styles/main.scss

```
@@ -1,3 +1,5 @@
-body {
-  margin: 0;
-}
+@import "./blocks/normalize.scss";
+
+@import "./blocks/utils/variables.scss";
+@import "./blocks/utils/mixins";
+@import "./blocks/calendar.scss";
```

```suggestion
@import "./utils/variables.scss";
@import "./utils/mixins";
@import "./normalize.scss";
@import "./blocks/calendar.scss";
```

Use proper order and file structure.
You have only one block - calendar.